Assets/AppCenter/AppCenterSettings.cs

```diff
@@ -48,7 +47,9 @@ public class AppCenterSettings : ScriptableObject
 
     public LogLevel InitialLogLevel = LogLevel.Info;
 
-    public UpdateTrack UpdateTrack = UpdateTrack.Public;
+    [CustomDropDownProperty("Public", 1)]
```

Comment:
Don't we need to start the count from 0?

Reply:
I guess it's just mapping to underlying API

Reply:
I want to use here 'UpdateTrack' values to use them later and reduce manipulations like '+-1'.